lib/application.ts

```diff
@@ -1381,6 +1381,11 @@ export class SNApplication {
     return this.protocolService!.repersistAllItems();
   }
 
+  public setEphemeralPersistencePolicy(): Promise<void> {
```

Comment:
Setters usually take an argument no? May make more sense to call this `setEphemeralPersistencePolicy`

Reply:
Oh yeah that's much better